### ISSUE TITLE
Screen reader does not read characters typed into quickpick

### ISSUE THREAD

**Issue by weinand** [microsoft/vscode#93932]
Refs #93648 
Orca 3.28.0 on Ubuntu 18.04.4 LTS

- press F1
- type into Quickpicks text box

Screen reader does not read typed characters (but it does read the characters that get deleted)

[I have no experience with screen readers. I just report what I noticed.
Someone who has to rely on a screen reader should test this and provide feedback.]

**Comment by isidorn** [comment]
I actually can not reproduce this. All the characters I type get nicely read out on Linux.
Might this be configurable by Orca?
Let's wait and see what @jvesouza and @joanmarie have to say

@chmarti unassigning you for now

**Comment by joanmarie** [comment]
@weinand  Chromium (and all products based on it) did not used to be accessible to Orca at all. Thus Orca had no support for Chromium until that changed. You should be using *at least* Orca v3.34.1 and *ideally* at least Orca v3.36.1.

**Comment by jvesouza** [comment]
@weinand I believe the problem is related to the version of the orca used. I tested in my environment without problems.

**Comment by weinand** [comment]
I'm neither a linux nor a Orca user. The test item did not specify what version of Orca to use.
I've tried to build Orca following [these steps](https://github.com/microsoft/vscode/issues/90446#issuecomment-585782600) but that failed (and I've spent already way too much time on this).
I suggest that someone with the correct Orca version tests this.

**Comment by isidorn** [comment]
Ok thanks @weinand
Thus closing this as invalid since you did not have a correct Orca version
